root/static/scripts/common/components/TagEditor.js

```diff
@@ -10,9 +12,15 @@ const React = require('react');
 const ReactDOM = require('react-dom');
 
 const {l, lp} = require('../i18n');
+const MB = require('../MB');
 const request = require('../utility/request');
+const TagLink = require('./TagLink');
 
-var Tag = Immutable.Record({tag: '', count: 0, vote: 0});
+const newTag: RecordFactory<UserTagT> = Immutable.Record({
```

Comment:
The docs for typing Records I used are here: https://github.com/facebook/immutable-js/blob/d23746f44ffc025d7bdcee7d7bb79c37d3dc3ea5/type-definitions/Immutable.d.ts#L2239